assets/scripts/cart.js

```diff
@@ -0,0 +1,200 @@
+import { create, main } from "./aggregate.js";
+
+function cart() {
+  const main = document.querySelector(".wrapper");
+  const fragment = document.createDocumentFragment();
+  const cartField = create("div", "cartField", fragment, "");
+  cartField.id = "cartField";
+  cartField.classList.add("droppable");
+  const list = create("ul", "list", cartField);
+  const total = create("div", "liRow", cartField);
+  total.classList.add("total");
+  if (document.title === "Bookshop") {
+    const confirmOrder = create(
+      "a",
+      "confirmOrderBtn",
+      cartField,
+      "Confirm order"
+    );
+    confirmOrder.classList.add("btn");
+    confirmOrder.href = "form.html";
+  }
+
+  main.append(fragment);
+}
+
+
+
+function addToCart() {
+  const buttonsAdd = document.querySelectorAll(".btnAdd");
+  const list = document.querySelector(".list");
+
+  buttonsAdd.forEach((button) =>
+    button.addEventListener("click", (event) => {
+      handleClick(event);
+    })
+  );
+
+  function handleClick(event) {
+    event.stopPropagation();
+    const button = event.currentTarget;
+    pushItem(button);
+  }
+  list.addEventListener("click", function (e) {
+    if (e.target.matches("button")) {
+      deleteItem(e);
+    }
+  });
+
+  list.addEventListener("itemsUpdated", mirrorToLocalStorage);
```

Comment:
It's better to rewrite the lines 49-52 as one listener and one function
```suggestion
  list.addEventListener("itemsUpdated", ()=>{
    mirrorToLocalStorage(),
    displayItems(),
    displayTotal(),
    disableConfirmBtn()
  });
```
Before
![image](https://user-images.githubusercontent.com/22891941/180012584-8a168c58-ea32-47d5-b269-a7c037e860f2.png)
After
![image](https://user-images.githubusercontent.com/22891941/180013271-e3a43ac5-99ba-40aa-b93b-df1dc779669b.png)